velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -107,6 +109,16 @@ TEST_P(MultiAggregatesTest, randomInput) {
   testWindowFunction(vectors, overClauses);
 }
 
+TEST_P(MultiAggregatesTest, basicRangeFrames) {
```

Comment:
Lets add atleast one test using kSortOrderBasedOverClauses.

Reply:
Done, thanks for the suggestion!